_pydevd_bundle/pydevd_comm.py

```diff
@@ -113,19 +113,19 @@
 from _pydevd_bundle.pydevd_comm_constants import *  # @UnusedWildImport
 
 # Socket import aliases:
-AF_INET, SOCK_STREAM, SHUT_WR, SOL_SOCKET, SO_REUSEADDR, IPPROTO_TCP, socket = (
+AF_INET, SOCK_STREAM, SHUT_WR, SOL_SOCKET, IPPROTO_TCP, socket = (
     socket_module.AF_INET,
     socket_module.SOCK_STREAM,
     socket_module.SHUT_WR,
     socket_module.SOL_SOCKET,
-    socket_module.SO_REUSEADDR,
     socket_module.IPPROTO_TCP,
     socket_module.socket,
 )
 
 if IS_WINDOWS and not IS_JYTHON:
     SO_EXCLUSIVEADDRUSE = socket_module.SO_EXCLUSIVEADDRUSE
-
+elif not IS_WASM:
```

Comment:
Actually I think this should likely be an if condition. Otherwise if windows won't have SO_REUSEADDR